pkg/sql/randgen/schema.go

```diff
@@ -331,7 +331,10 @@ func PopulateTableWithRandData(
 				// them to the list of columns to insert data into.
 				continue
 			}
-			colTypes = append(colTypes, tree.MustBeStaticallyKnownType(col.Type.(*types.T)))
+			if _, ok := col.Type.(*types.T); !ok {
+				return 0, errors.Newf("No type for %v", col)
```

Comment:
Why was this change necessary?

Reply:
iirc this was panicking on user-defined types without a helpful error message.

Reply:
Thanks for the explanation.